TABTestKit/Classes/Extensions/XCTestCase+Screenshot.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  XCTestCase+Screenshot.swift
+//  TABTestKit
+//
+//  Created by Stefan Renne on 09/11/2020.
+//
+
+import XCTest
+
+extension XCTestCase {
+
+    func attachScreenshot() {
+        // if tests have no host app there is no point in making screenshots
```

Comment:
When would UI tests not have a host app?

Reply:
Removed it